compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs

```diff
@@ -105,7 +105,12 @@ createProjectPackageDb opts thisSdkVer deps0 dataDeps = do
             (pkgId, package) <-
                 either (fail . DA.Pretty.renderPretty) pure $
                 Archive.decodeArchive Archive.DecodeAsMain dalf
-            pure (pkgId, package, dalf, stringToUnitId name)
+            pure
+                ( pkgId
+                , package
+                , dalf
+                , stringToUnitId $
```

Comment:
This is duplicated between here and `Damlc.hs`. It would be good to factor this out into something like `getUnitId :: String -> PAckageId -> UnitId`.

Reply:
done, I put it together with`stripPkgId` in the `LF.Reader` module.